README.md

```diff
@@ -50,6 +50,20 @@ After provision beanstalk console will be available at [http://localhost:7654](h
 
 [Download](https://github.com/ptrofimov/beanstalk_console/archive/master.zip), unzip files to your *www* directory and launch from *public* directory, enjoy!
 
+### Run as a Docker container
+
+Install [Docker](https://docs.docker.com/installation/) then build and run with the following command (from project root):
+
+    docker build --rm -t beanstalk_console .
+    docker run -d -p "80:80" --name beanstalk_console beanstalk_console
+
+If you would rather just run the existing automated build of this project, run (from project root):
+
+    docker run -d -p "80:80" --name beanstalk_console agaveapi/beanstalkd-console
```

Comment:
The [Agave API](http://agaveapi.co) is open API platform providing science-as-a-service to the global research community. We use beanstalkd and your console in our devops infrastructure. All of our Docker images are available on the hub and customized to be easy to use and include in automated developer workflows. If this project gets a Docker repo for an official automated build, the link should probably be updated. As it stands now, few people build their 3rd party images from scratch. Prebuilt images are easier for people to pull and try out.